examples/gitlab/README.md

```diff
@@ -0,0 +1,229 @@
+# GitLab Application Example
+
+This example assumes you already have EveryAuth configured in your development environment. In case you don’t, follow the [configuration steps](https://github.com/fusebit/everyauth-express#getting-started).
+
+You have an existing Express.js application that needs to integrate with GitLab API to display the following information:
+- User profile information
+- User starred repositories
+
+The application will display the authorizing user’s GitLab profile that looks like the following:
+
+![Screenshot demo](blog-using-gitlab-with-everyauth.png "Screenshot demo")
+
+### Configuring EveryAuth
+
+This blog post assumes you already have EveryAuth configured in your development environment. In case you don’t, follow the [configuration steps](https://github.com/fusebit/everyauth-express#getting-started) from the EveryAuth GitHub Repository.
+
+A basic Express application will look like the following:
+
+```javascript
+const express = require('express');
+
+const app = express();
+const port = 3000;
+
+app.listen(port, () => {
+  console.log(`Example app listening on port ${port}`);
+});
+```
+
+Let’s add support to EveryAuth and configure the Gitlab service so we can interact with their API.
+
+## Install dependencies
+
+For interacting with Gitlab API from Node.js we will use the popular Node.js SDK for GitLab [@gitbeaker/node](https://www.npmjs.com/package/@gitbeaker/node)
+
+```shell
+npm i @gitbeaker/node
+```
+
+Install the [@fusebit/everyauth-express](https://www.npmjs.com/package/@fusebit/everyauth-express) package
+
+```shell
+npm i @fusebit/everyauth-express
+```
+
+Install the [uuid](https://www.npmjs.com/package/uuid) package. Used to generate a unique identifier for the userId.
+
+```shell
+npm i uuid
+```
+Install the [cookie-session](https://www.npmjs.com/package/cookie-session) package to allow your application to establish a session (an HTTP-Only cookie) to store the authorizing user id.
+
+```shell
+npm i cookie-session
+```
```

Comment:
All of this should be just part of the package.json and not called out separately.  The session stuff is just there, for example, as illustrative purposes.

Reply:
Fixed